pkgs/development/haskell-modules/configuration-hackage2nix.yaml

```diff
@@ -67,6 +67,8 @@ core-packages:
 # comment saying "# LTS Haskell x.y". Any changes after that commend will be
 # lost the next time `update-stackage.sh` runs.
 default-package-overrides:
+  # This was only intended for ghc-7.0.4, and has very old deps, one hidden behind a flag
+  - MissingH ==1.4.2.0
```

Comment:
I don't understand the purpose of this addition. We have `MissingH-1.4.2.1` by default. Why is version `1.4.2.0` superior? As far as I can tell, the older version does not compile with ghc-8.8.2 either: https://matrix.hackage.haskell.org/#/package/MissingH

Reply:
*As I laid out in the commit message for the change*: 1.4.2.0 is superior because it doesn't depend on a version of `unix` that isn't available, and can't be jailbroken because `jailbreak-cabal` lacks the ability to remove version bounds that are hidden behind flags.

If you look at the actual changelog for 1.4.2.1 on hackage (https://hackage.haskell.org/package/MissingH-1.4.2.1/changelog), you can see that it specifically says:
```
1.4.2.1

    GHC 7.0 only compat release
```
With regards to whether or not it compiles...I tested it, it compiled (and allowed `ConfigFile` to compile as well).  I don't know what else to say?